tests/acceptance/features/users.feature

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+Feature: Get the list of users from the IdM.
+
+    Scenario:  01: Get the list of trial users
+      Given a user with name "trial1", password "trial" and role "trial"
+      Given a user with name "trial2", password "trial" and role "trial"
+      Given a user with name "basic1", password "trial" and role "basic"
+      When I request a list of trial users from the Keystone
+      Then the Keystone returns a list with trial "2" users
+
+    Scenario: 02: Get the list of community users
+      Given a user with name "community1", password "password" and role "community"
+      Given a user with name "community2", password "password" and role "community"
+      Given a user with name "basic1", password "password" and role "basic"
+      When I request a list of community users from the Keystone
+      Then the Keystone returns a list with community "2" users
+
+    Scenario Outline: 03: Get list of several users
+      Given a user with name "<username>", password "<password>" and role "<role>"
+      When I request a list of trial users from the Keystone
+      Then the Keystone returns a list with trial "<listtrial>" users
```

Comment:
I think that you want to say a number of trial users equal to X, the same for community.